renovate.json

```diff
@@ -14,5 +14,13 @@
       "depNameTemplate": "sgerrand/alpine-pkg-glibc",
       "versioningTemplate": "loose"
     }
-  ]
+  ],
+  "packageRules": [
+    {
+      "matchPackagePatterns": [
+        "^grpc"
+      ],
+      "groupName": "grpc"
+    }
+  }
```

Comment:
It seems invalid JSON

Reply:
https://github.com/seqsense/protoc-assets/pull/12/commits/2c319c6d1579836c6457e0267c05efb3a121f469 fixed the json